articles/quickstart/spa/angular2/_includes/_centralized_login.md

```diff
@@ -2,7 +2,34 @@
 
 ## Create an Authentication Service
 
-Create a service to manage and coordinate user authentication. You can give the service any name. In the examples below, the service is `AuthService` and the filename is `auth.service.ts`.
+As a first step, create an `auth` sub-directory within your `src` directory. If you created your project using the Angular CLI, a good location for the `auth` sub-directory is `src/app`. 
+
+Next, inside the `auth` sub-directory, create a file named `auth0-variables.ts` that will store the Auth0 application keys and callbackURL constants that are going to be used to communicate with Auth0 through an Angular service.
+
+```ts
```

Comment:
I think it's a good idea to have it. It is a good practice to keep a separate file for these type of constants in Angular. If the services need to be refactored or changed, the constants remain in one easy to find place. This also makes testing easier since the variables can be mocked - for whatever reason needed.